packages/ui/stories/dao/DaoCard.stories.tsx

```diff
@@ -10,7 +10,25 @@ export default {
 const Template: ComponentStory<typeof DaoCard> = (args) => <DaoCard {...args} />
 
 export const Default = Template.bind({})
-Default.args = {}
+Default.args = {
+  name: 'Modern DAO',
+  description:
+    'This approach allows us to implement a completely custom component design without writing a single line of custom CSS.',
+  imageUrl: '/placeholders/1.svg',
+  href: '/',
+  established: '14 May, 2022',
```

Comment:
hey, what do you think of using DateTimeFormat for rendering dates https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat#examples 

checkout the way i did it (actually curious on your thoughts) https://github.com/DA0-DA0/dao-dao-ui/pull/835/files#diff-ff847c18be1bc25afc33a85049069efab4aee0876b6c69f33ae27ef202961276R61

i was thinking we should create functions in i18n for general date formatting we do

Reply:
ah thanks for reading / thinking about this! I like your approach here of taking a `Date` as an argument - lets us style consistently. wIll update this. :)